components/annot-discrete-choices.vue

```diff
@@ -130,11 +150,11 @@
 
                 const removedColumns = p_oldColumns.filter(column => !p_newColumns.includes(column));
 
-                if ( removedColumns.length > 0 ) {
+                if (removedColumns.length > 0) {
```

Comment:
Hhm. Here is the problem with this: this rule is not part of the current eslint rules. We already have 12 additional rules defined. This particular rule does not seem to have an exact correspondence among eslint rules, so we may have to set several additional rules. I'm against coming up with our own, custom neurobagel styleguide - that seems like a lot of work for something relatively unimportant. But we still want to make sure that we avoid all of this whitespace diff stuff. So I think we have two options:

1. Pick any existing style guide that is sufficiently restrictive to cover any white-space diff cases (e.g. google, airbnb) and stick with that. That would also mean removing our own exceptions to this style guide.
2. setting up prettier again as a code formatter to actively enforce certain style rules that the linter may not care about. 

Between the two, I prefer 1 because 2 will be additional work to set up. In both cases we'd have to check that this actually solves the white-space around parantheses case. 

What do you think @jarmoza ?